lib/process-services/src/lib/process-list/components/start-process.component.ts

```diff
@@ -214,6 +230,48 @@ export class StartProcessInstanceComponent implements OnChanges, OnInit, OnDestr
             });
     }
 
+    loadAppsAndStartProcess() {
+        this.appsProcessService
+            .getDeployedApplications()
+            .pipe(map((response: AppDefinitionRepresentationModel[]) => {
+                    let currentApplication: AppDefinitionRepresentationModel;
+                    let applications: AppDefinitionRepresentationModel[] = [];
+
+                    applications = this.removeDefaultApps(response);
+
+                    if (applications && applications.length > 0) {
+                        currentApplication = applications[0];
+                        if (this.appId) {
+                            const filteredApp = applications.find( app => app.id === +this.appId );
+                            currentApplication = filteredApp ? filteredApp : applications[0];
+                        }
```

Comment:
```
 if (applications && applications.length > 0) {
                const filteredApp = applications.find( app => app.id === +this.appId );
                currentApplication = filteredApp ? filteredApp : applications[0];
}
```

I think this simplified code can give you the same result. In case appId is not defined the .find will return undefined and because of this code `filteredApp ? filteredApp : applications[0]; `you will get the first selected by default

Reply:
Updated